rpc/codec.go

```diff
@@ -1,28 +0,0 @@
-// Copyright 2017 Monax Industries Limited
-//
```

Comment:
It's not general in the sense of being used by both RPCs. I don't that we'd particularly likely to use it for the next iteration of the RPC